.eslintrc.json

```diff
@@ -1,150 +1,162 @@
 {
   "root": true,
-  "parser": "@typescript-eslint/parser",
-  "parserOptions": { "project": "./tsconfig.json" },
-  "extends": [
-    "./node_modules/gts",
-    "plugin:import/errors",
-    "plugin:import/warnings",
-    "plugin:import/typescript",
-    "plugin:n/recommended"
-  ],
   "env": {
     "browser": true,
     "node": true
   },
-  "plugins": ["node", "ban", "import", "deprecation", "gpuweb-cts", "n"],
-  "rules": {
-    // Core rules
-    "linebreak-style": ["warn", "unix"],
-    "no-console": "warn",
-    "no-throw-literal": "warn",
-    "no-undef": "off",
-    "no-useless-rename": "warn",
-    "object-shorthand": "warn",
-    "prefer-promise-reject-errors": "warn",
-    "quotes": ["warn", "single", { "avoidEscape": true, "allowTemplateLiterals": true }],
+  "extends": ["./node_modules/gts"],
+  "overrides": [
+    {
+      "files": ["**/*.ts"],
```

Comment:
Moving all this config into the "overrides" section lets us say that it applies only to `.ts` files. This _should_ make it so the default "gts" configs are used for `.js` files.